flare_portal/experiments/models/core.py

```diff
@@ -37,6 +37,8 @@ class Experiment(models.Model):
     project = models.ForeignKey("experiments.Project", on_delete=models.CASCADE)
     trial_length = models.FloatField()
     rating_delay = models.FloatField(default=1)
+    iti_min_delay = models.PositiveIntegerField(default=1, verbose_name="ITI Min Delay")
+    iti_max_delay = models.PositiveIntegerField(default=3, verbose_name="ITI Max Delay")
```

Comment:
`verbose_name` generally is lowercase, except for instances like this where it's using an acronym. So maybe better to type `ITI min delay` so the front-end can control whether or not to title case or not.

Reply:
Ah thanks!